### DISCUSSION TITLE
feature: Define the location of the well screen in Flopy

### DISCUSSION THREAD

**Discussion by wsc1008-cugb** [modflowpy/flopy#1512] [General]
Hi，I‘m try to add _"screen top’"_ and _"screen bottom"_ of wells in my MT3DMS model(like the pic following), but I didn't find the corresponding parameters in WEL package. And it can be realized in GMS. SO how could I define the **_screen_** in Flopy.
The other problem is: If the location(top and bottom) of the srcreen is inside the cell, which means the screen only occupy part of the cell. How to distribute the flow? Thank you！
![NNMY@X XFBJ}%8GECH1E$6D](https://user-images.githubusercontent.com/74029851/185783678-07e66474-1b6a-4b0d-97fb-e3d414fa074b.png)

**Comment by ashafarsana** [comment]
I also want to know how to assign screen top and bottom in Flopy.

**Comment by dbrakenhoff** [comment]
This is not really a bug, or feature request, more a user question, which I think is what StackOverflow is used for (or GH discussions?).

Having said that, perhaps the following links can be of help, about the MNW package which allows setting of top and bottom of well screens:
- https://pubs.usgs.gov/tm/tm6a30/pdf/TM-6A30_hirez.pdf
- https://github.com/modflowpy/flopy/blob/develop/examples/Notebooks/flopy3_mnw2package_example.ipynb
